app/src/main/shared/components/Page.tsx

```diff
@@ -16,13 +16,19 @@ export function Page<TState, TLocationProps>(pageActionCreators: AbstractPageAct
         }
     };
 
+    const mapStateToProps = (state: TState): Partial<PageProperties<TLocationProps>> => {
+        return {
+            title: pac.title(state)
+        }
+    };
```

Comment:
Is there a test for this bit?

Reply:
Ah, not in the generic tests, good catch